components/infobox/wikis/ageofempires/infobox_person_player_custom.lua

```diff
@@ -344,6 +354,7 @@ function CustomPlayer._getLatestPlacement(game)
 		CustomPlayer._buildPlacementConditions(),
 		ConditionNode(ColumnName('game'), Comparator.eq, game)
 	}
+	mw.log(conditions:toString())
```

Comment:
log intended?

Reply:
```suggestion
```
Totally intended, not just leftover debugging